### ISSUE TITLE
整体架构

### ISSUE THREAD

**Issue by Wang-1812** [wurenkai/MHA-UNet#2]
论文中是不是写错啦，原本的MHA-Unet到下面怎么变成介绍MSA-Unet啦，图还是MHA-Unet的图

**Comment by wurenkai** [comment]
Yes, MSA-UNet should all be MHA-UNet, this was a spelling error. We have corrected this in the review version of the submitted journal and will be updating it in the near future.
